edi/security/ir_model_access.xml

```diff
@@ -83,7 +83,7 @@
     <record id="rule_edi_exchange_record_manager" model="ir.rule">
         <field name="name">Manager EDI exchange records</field>
         <field name="model_id" ref="edi.model_edi_exchange_record" />
-        <field name="domain_force">[('1', '=', '1')]</field>
+        <field name="domain_force">[(1, '=', 1)]</field>
```

Comment:
it should work anyway no? The point is that the domain should always be TRUE.

Reply:
As it is an string, it tries to use the field names '1', and it obiously does not exist.